src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -142,6 +142,14 @@ public boolean hasSession(Session session) {
         return students.hasSession(session);
     }
 
+    /**
+     * Returns true if session {@code Session} overlaps with any session belonging to any student
+     * in the unique student list
+     */
+    public boolean hasOverlappingSession(Session session) {
+        return students.hasOverlappingSession(session);
+    }
+
     //// util methods
```

Comment:
Actually outside of scope, but this comment here seems abit unnecessary with ////, I think we could just use // instead, so that it won't be flagged as a "bug" during PE dry run.

Reply:
Were this from AB3? Okay I'll remove the extra backslashes